lib/rust/ensogl/component/scroll-area/src/lib.rs

```diff
@@ -16,14 +16,17 @@
 #![warn(unused_import_braces)]
 #![warn(unused_qualifications)]
 
+use ensogl_core::display::shape::*;
 use ensogl_core::prelude::*;
 
 use enso_frp as frp;
 use ensogl_core::application::Application;
 use ensogl_core::control::callback;
 use ensogl_core::control::io::mouse;
+use ensogl_core::data::color;
 use ensogl_core::display;
 use ensogl_core::display::object::ObjectOps;
+use ensogl_core::display::scene::layer::MaskedLayer;
```

Comment:
As described in style guide, prefer to use qualified usage. The usage should be `layer::Masked`.

Reply:
Done